src/app/routes.tsx

```diff
@@ -45,19 +47,25 @@ const routes: AppRouteConfig[] = [
     component: <AWSImageBrowser title='Cloud Image Directory | AWS Image Browser'/>,
     exact: true,
     label: 'AWS',
-    path: '/browser/AWS',
+    path: '/browser/aws',
```

Comment:
Nice, thanks for fixing this.
This always annoyed me and I always forgot to fix it in the end